server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/SPF.java

```diff
@@ -91,11 +100,12 @@ public void setSPFDnsService(org.apache.james.jspf.core.DNSService spfDnsService
     public void init() {
         debug = Boolean.parseBoolean(getInitParameter("debug", "false"));
         addHeader = Boolean.parseBoolean(getInitParameter("addHeader", "false"));
+        addHeader = Boolean.parseBoolean(getInitParameter("checkLocalIps", "false"));
```

Comment:
Q: Should not this be a new boolean variable rather than override the `addHeader`?

BTW likely a bit of docs should be added for the new property IMO.

Reply:
Likely a typo

Reply:
Good catch, should be fixed